templates/Park/show.html.twig

```diff
@@ -5,7 +5,7 @@
 {% set subTitle = park.country.name|trans([], 'database') %}
 {% block title %}{{ title }} - {{ subTitle }}{% endblock %}
 
-{% set isImperial = units.isImperial(app.request.locale) %}
+{% set isImperial = units.isImperial(app.request.locale, app.user) %}
```

Comment:
To be removed when dependency injection is fixed

Reply:
Same